### ISSUE TITLE
Depiction of multifragment mol-file causes overlaps

### ISSUE THREAD

**Issue by fredrikw** [openbabel/openbabel#1584]
- [x] I believe this to be a bug with Open Babel
- [ ] This is a feature request

## Environment Information
Open Babel version: Trunk (with addition of PR #1582)
Operating system and version: OSX 12.5

## Expected Behavior
Depiction of a molfile containing ZnCl2 ([Zn+2].[Cl-].[Cl-]) (content of molfile:

  ChemDraw06021710362D

  3  0  0  0  0  0  0  0  0  0999 V2000
   -0.2736    0.3162    0.0000 Cl  0  5  0  0  0  0  0  0  0  0  0  0
    0.2736    0.3162    0.0000 Zn  0  2  0  0  0  0  0  0  0  0  0  0
    0.1911   -0.3162    0.0000 Cl  0  5  0  0  0  0  0  0  0  0  0  0
M  CHG  3   1  -1   2   2   3  -1
M  END
) should keep the coordinates from the file in the resulting image.
![zncl2_chemdraw](https://cloud.githubusercontent.com/assets/86855/26718254/ab9aa108-4780-11e7-88ea-36e0949cd7df.png)

## Actual Behavior
The resulting image gets all atoms overlayed.
![zncl2_openbabel](https://cloud.githubusercontent.com/assets/86855/26718289/ce5157aa-4780-11e7-91f3-05b1aa8180e7.png)

## Steps to Reproduce
1. obabel -imol -O ~/Desktop/ZnCl2_openbabel.png
2. paste the above file content
3. look at resulting image

**Comment by fredrikw** [comment]
I found the reason and the problem only occured when there were no bonds in the molecule. The current scaling was based on average bond length and set to 1 if no bonds were present. That meant that depending on the coordinates used in the original file, the fragments could be almost on top of each other (or separated by a huge amount). I created a PR (#1585) to use the minimum atom distance instead of average bond length. There might be a better option, but this seems to work.
